changelogs/fragments/26-remove_tls_requirements.yml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
+- mysql_user - Correct procedure to check existing TLS requirements.
+- mysql_user - Minor syntax fixes.
+improvements:
+- mysql_user - Added tests to verify that TLS requirements are removed with an empty `tls_requires` parameter.
+- mysql_user - Added examples on how to remove TLS requirements by setting an empty `tls_requires` parameter.
+- mysql_user - Refactored TLS requirement tests into their own file.
```

Comment:
```suggestion
```
we don't need fragments for doc changes and tests, it's necessary only for things that can affect module/plugin behavior